regression-test/suites/query_p0/cross_db/cross_db.groovy

```diff
@@ -54,7 +54,7 @@ suite("cross_db") {
         (4, 40)
     """
 
-    sql """SELECT * FROM cross_db_1.cnt_table,cross_db_2.cnt_table"""
+    sql """SELECT * FROM cross_db_1.cnt_table as x,cross_db_2.cnt_table as y"""
```

Comment:
why change this case? `SELECT * FROM cross_db_1.cnt_table,cross_db_2.cnt_table` should not throw exception

Reply:
i will fix it